### DISCUSSION TITLE
the esp32 does not map the buttons

### DISCUSSION THREAD

**Discussion by khtp** [darthcloud/BlueRetro#223] [Q&A]
hello good morning or good evening If I leave it by default, the xbox menu button does not work as the start button, the view button does not respond and neither does the xbox button, but if I map those buttons, for example, the xbox button as start, it does map the controls but the rest is misconfigured or not another button stops working for example x. If I map xyab from xbox to the mapping of the gamecube xyab, that is, they are identical if it works for me but the x button stops working and the view button becomes the z button the rest of the buttons work fine for me I am using version 1.2.1 and tried with 1.5 but nothing. It is configured for gamecube
![275242478_486881219583489_386690870840700878_n](https://user-images.githubusercontent.com/50355226/158100353-ba57cf16-086b-498f-897e-d274aecc05b2.jpg)

**Comment by darthcloud** [comment]
BlueRetro only support latest xbox Firmware, update with Update via the [Xbox accessories Win10 app](https://www.microsoft.com/en-us/p/xbox-accessories/9nblggh30xj3).

**Reply by khtp** [reply]
it won't let me open it because it asks me to have the latest windows update but I can't because my laptop doesn't support 11, I'm on windows 10 21h2, 'is there another way to get that program?

**Reply by darthcloud** [reply]
Pretty sure, having latest windows update does not means installing win 11. Its a separate process.

**Reply by khtp** [reply]
good afternoon, a question does it work on ps2 model fat (large) first model sphc 30001 because i tried it and it didn't work but on a slim sphc 7000x this if i detect it, could it be that my ps2 fat connectors will have a short? can it be repaired? the driagram is based on other youtube channels  
![image](https://user-images.githubusercontent.com/50355226/158962759-c4b04df4-3d4f-49f9-92d9-6d0b0695963b.png)

**Comment by khtp** [comment]
![IMG-20220313-WA0005](https://user-images.githubusercontent.com/50355226/159044901-a8cfc671-061b-44f6-8477-1b125f493fb2.jpg)
